docs/modules/user-guide/partials/proc_adding-a-command-section-to-a-devfile.adoc

```diff
@@ -0,0 +1,75 @@
+[id="proc_adding-a-command-section-to-a-devfile_{context}"]
+= Adding a command section to a devfile
+[role="_abstract"]
+
+Specify the commands you want to run in your workspace.
+
+.Prerequisites
+
+* xref:adding-schema-version-to-a-devfile.adoc[]
+* xref:adding-a-name-to-a-devfile.adoc[]
+* xref:adding-projects-to-a-devfile.adoc[]
+
+.Procedure
+
+. Add a `commands` section to a devfile that contains a list of one or more commands.
+
+. For each command, define a unique value for the mandatory `id` attribute.
+
+. For each command, define a mandatory type of one of the following types:
+
+* `exec`
+* `apply`
+* `composite`
+
++
+.Sample command
+[source,yaml]
+----
+ commands:
+   - id: build
+     exec:
+      component: mysql
+      commandLine: mvn clean
+      workingDir: /projects/spring-petclinic
+----
+
+. Define attributes for the `exec` command to run using the default shell in the container.
++
+ * A `commandLine` attribute that is a command to run.
+ * A `component` attribute that specifies the container in which to run the command.
```

Comment:
The `component` attribute value must correspond to an existing container component name